books/RayTracingInOneWeekend.html

```diff
@@ -466,12 +466,21 @@
 that returns the color of the background (a simple gradient).
 
 I’ve often gotten into trouble using square images for debugging because I transpose $x$ and $y$ too
-often, so I’ll stick with a 200×100 image. I’ll put the “eye” (or camera center if you think of a
-camera) at $(0,0,0)$. I will have the y-axis go up, and the x-axis to the right. In order to respect
-the convention of a right handed coordinate system, into the screen is the negative z-axis. I will
-traverse the screen from the lower left hand corner, and use two offset vectors along the screen
-sides to move the ray endpoint across the screen. Note that I do not make the ray direction a unit
-length vector because I think not doing that makes for simpler and slightly faster code.
+often, so I’ll use a non-square image. For now we'll use a 16:9 aspect ratio, since that's so
+common.
+
+In addition to setting up the pixel dimensions for the rendered image, we also need to set up a
+virtual viewport through which to pass our scene rays. For the standard square pixel spacing, the
+viewport's aspect ratio should be the same as our rendered image. We'll just pick a viewport two
+units in height. Ultimately, changing the scale of the viewport (while holding the focal distance
```

Comment:
focal d hasn't been intro'd yet, complicates

Reply:
Agree with David. Cut this sentence. If the user wants to play with the height, they'll intuit how focal length and zoom interact.

Reply:
Also, you have it written `focal_distance` here when it's `focal_length` in the code below. I'm pretty sure those are the same thing. If this sentence is kept, those should be synced.

Reply:
The book has confusing terminology between "focus distance" and "focal distance" (or "focal length", which I think I prefer). The focus distance is the virtual plane at which objects appear in perfect focus. The focal length is the distance between the projection point and the projection plane. These are very different (in fact, they _must_ be).

I'll clarify the text.

Reply:
Yeah, sorry, I just realized my mistake as well

Reply:
I think we should keep things as `focal length` and `focus distance`.

Reply:
I need to be more careful when flinging terms.

Reply:
Naaaaaaah